manifests/sqlitebrowser/sqlitebrowser/3.11.2.yaml

```
@@ -0,0 +1,17 @@
+Id: sqlitebrowser.sqlitebrowser
+Version: 3.11.2
+Name: DB Browser for SQLite
+Publisher: sqlitebrowser
+License: GNU GPL 3
+LicenseUrl: https://github.com/sqlitebrowser/sqlitebrowser/blob/master/LICENSE
+AppMoniker: sqlitebrowser
+Tags: sqlite, dbbrowser, database
+Description: DB Browser for SQLite (DB4S) is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite.
+Homepage: https://sqlitebrowser.org/
+Installers:
+  - Arch: x64
+    Url: https://download.sqlitebrowser.org/DB.Browser.for.SQLite-3.11.2-win64.msi
```

```suggestion
    Url: https://github.com/sqlitebrowser/sqlitebrowser/releases/download/v3.11.2/DB.Browser.for.SQLite-3.11.2-win64.msi
```

Same installer but from GitHub, which does support straight GET results